StortSpelProjekt/Game/src/Gamefiles/Components/UpgradeComponents/Upgrades/UpgradePotionOfStrength.cpp

```diff
@@ -0,0 +1,66 @@
+#include "UpgradePotionOfStrength.h"
+#include "EngineMath.h"
+#include "ECS/Entity.h"
+
+UpgradePotionOfStrength::UpgradePotionOfStrength(Entity* parent) : Upgrade(parent)
+{
+	// Set the name of the upgrade
+	SetName("PotionOfStrength");
```

Comment:
Please add "Upgrade" to the begining of the name so it follows the naming convention of the other upgrades. Needed since this is built upon by other parts of our game such as UpgradeGUI.